status/statistics.py

```diff
@@ -239,6 +241,13 @@ def get(self):
                 self.flowcell_aggregates[fa][1],
                 self.cleaning,
             )
+        for fa in self.nanopore_flowcell_aggregates:
+            data[fa] |= get_stats_data(
```

Comment:
Do you mean to do a bitwise OR to this assignment (`|=`)?

Reply:
Sort of, for dictionaries the `|` stands for union instead of or... So `a |= b` translates to `a = a | b` which is to assign the union of a and b to a. Maybe that qualifies to have a comment in the code?

Reply:
I just added a comment to try to explain this in the code.

Reply:
Ah the joys of dynamic typing. That would make more sense then 😉